prowler/providers/aws/aws_provider.py

```diff
@@ -106,9 +107,13 @@ def __init__(self, arguments: Namespace):
             self.session.current_session.region_name, input_regions
         )
 
-        caller_identity = validate_aws_credentials(
+        connected, caller_identity = self.test_connection(
             self.session.current_session, sts_region
         )
+        # TODO: review this for CLI and SDK usage
+        if not connected:
```

Comment:
I would raise a critical error here.

Reply:
Or maybe not using the boolean, only for the API.

Reply:
The <test_connection> raises an ERROR log, I think with that is more than enough. We need to think how to handle that for the CLI.

Reply:
why then using the `connected` attribute here?

Reply:
because we need to abort the provider's execution if it is not connected.